client/core/core.go

```diff
@@ -1385,6 +1388,24 @@ func (c *Core) loadWallet(dbWallet *db.Wallet) (*xcWallet, error) {
 	}
 	wallet.Wallet = w
 	wallet.connector = dex.NewConnectionMaster(w)
+
+	// Parse fee rate limit
+	// NOTE We parse fee rate limit 'feelimit' settings and store it on xcWallet
+	// unlike other settings which are being configured and set on each asset's
+	// ExchangeWallet instance (see above asset.Setup call) as it required on
+	// order placement time
+	feeLimit := walletCfg.Settings["feelimit"]
+	if feeLimit != "" {
+		floatLimit, err := strconv.ParseUint(feeLimit, 10, 32)
+		if err != nil {
+			return nil, err
+		}
+		wallet.feeRateLimit = uint32(floatLimit)
+	} else {
+		// If user didn't provide limit use default limit
+		wallet.feeRateLimit = defaultFeeRateLimit
```

Comment:
Currently `ConfigOpts` works just fine with empty default values, but you are right, using `ConfigOpts` defaults would make it easier to have asset specific defaults, actually that's was my first approach but then users will see defaults in the form instead of empty optional fields so I changed to empty by default and set value in core.

Anyway, will revert that to have asset specific `ConfigOpts` default without optional fields

Reply:
I agree. This whole thing is best done at the **core** level, but it's a lot more work. Can switch later. Wallet settings are easy.